addons/mail/static/src/core/common/composer.scss

```diff
@@ -131,37 +131,18 @@
 }
 
 .o-mail-Composer-html {
-    padding-top: 10px; // carefully crafted to have the text in the middle in chat window
-    padding-bottom: 10px;
-    padding-left: map-get($spacers, 1);
-    padding-right: map-get($spacers, 1);
-
-    line-height: 1.42857143 !important; // so that input is rounded to 20px = 14px (base font) * 1.42857143 (line-height)
-
-    .o-mail-Composer.o-editing & {
-        padding-left: map-get($spacers, 2);
-        padding-right: map-get($spacers, 2);
-    }
-
-    .o-mail-Composer.o-extended & {
-        padding-left: map-get($spacers, 2) + map-get($spacers, 1);
-        padding-right: map-get($spacers, 2);
-    }
+    padding-top: 11px;
 
     .o-mail-Composer.o-chatWindow & {
         padding-top: 7px;
-        padding-bottom: 7px;
-        padding-left: map-get($spacers, 1) / 2;
-        padding-right: map-get($spacers, 1) / 2;
     }
 
     .o-mail-Composer.o-isUiSmall & {
-        padding-top: 10px;
-        padding-bottom: 10px;
+        padding-top: 7px;
```

Comment:
Can move this change to text mode too, actually the `o-chatWindow` and `o-isUiSmall` should have same symmetric padding top/bottom of `7px;`. Right now the text composer is unbalanced (7/10px) so that's good to fix too.

Reply:
For clarity since the code preview doesn't help: this comment is on the specific html rule:
```scss
.o-mail-Composer-html {
   .o-mail-Composer.o-isUiSmall & {
}
```
That I suggest to remove and fix the `inputStyle` rule to have this balanced `7px` for top/bottom padding.